conduce/ingest.py

```diff
@@ -0,0 +1,28 @@
+from . import api
+from . import util
+
+
+def get_dataset_id(dataset_name, **kwargs):
```

Comment:
I don't really care for this function in a file called `ingest.py`.  It should either move to `api.py` or this filename should change.  I like the latter, but I can't think of a good name at this time.

Reply:
I suppose this function could be removed, but it provides a nice interface.  If `find_resource` had an option to return only the first result and not an array this function would not be necessary.  `find_resource(..., first=True).get('id')` would be equivalent